Wolox Tranining App/SignUpViewController.m

```diff
@@ -0,0 +1,71 @@
+//
+//  SignUpViewController.m
+//  Wolox Tranining App
+//
+//  Created by Francisco Depascuali on 4/22/15.
+//  Copyright (c) 2015 Francisco Depascuali. All rights reserved.
+//
+
+#import "SignUpViewController.h"
+#import "Toast/UIView+Toast.h"
+#import "SignUpViewModel.h"
+
+
+@interface SignUpViewController ()
+@property (strong, nonatomic) SignUpViewModel *viewModel;
+@end
+
+@implementation SignUpViewController
+
+- (IBAction)termsButtonClicked:(id)sender {
+    [self.viewModel openPage: @"http://www.wolox.com.ar"];
```

Comment:
Esta bien esto que hiciste, aunque la responsabilidad de saber a donde llevar cuando el usuario hace click en terms and conditions podria ser del view model.

Podrias tener un metodo en el view model que sea ```openTermsAndConditions``` que conozca la URL. Esto la ventaja que te da es que sacas reponsabilidad del view controller, la trasladas al view model, y lo podes testear.